src/components/Courses.jsx

```diff
@@ -19,6 +32,17 @@ const Courses = () => {
         });
   }, [URL]);
 
+  const handleClick = async (e, code) => {
+    e.preventDefault();
+    const response = await deleteCourse(code);
+    console.log(response);
+    if (response) {
```

Comment:
shouldn't you check the response status code or something else? `response` will be truthy even for a bad request

Reply:
I want to leave this for now because I want to test it to see what's in the response later

Reply:
Understood!

But, when committing to master, you always want it in a deployable state. So you don't want to commit your debugging logs. They are easy to add back later when you are testing/debugging on your own

Reply:
my comment was not about the log, but about the `if (response)`